app/assets/stylesheets/admin/calendar/_calendar.scss

```diff
@@ -171,4 +180,35 @@
       display: block;
     }
   }
+
+  #calendar-step-modal {
+    .modal-content {
+      background: none;
+      border: none;
+      box-shadow: none;
+    }
```

Comment:
A comment on this block would be appropriate. I didn't get initially why you were overriding these things.